extra/utils/permissions.js

```diff
@@ -0,0 +1,26 @@
+const permissions = {     //Permissions is a object having various arrays and it is used to check all operations a person can perform
+    'getUsers': {
+    all: ['head-trainer'],
+    read : ['trainee', 'trainer'],
+    write : ['trainer'],
+    delete: [],
+    }
+    }
+    function HasPermission(ModuleName,role,PermissionType)
```

Comment:
please have a look into the camel casing :
"HasPermission" - "hasPermission"

Reply:
Done